kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue

```diff
@@ -174,16 +174,16 @@
                 },
               });
             } else {
-              if (this.$route.params.quizId === exam.id) {
```

Comment:
I don't think we should remove this - but we should coerce them both to strings for the comparison because for draft exams the exam.id will be an integer, but the route parameter will be a string.

When I did a subsequent save, not having this here led me to get a redundant navigation error from the replace below.